confluent_kafka_helpers/message.py

```diff
@@ -20,18 +33,14 @@ def __repr__(self):
     def __bool__(self):
         return True if self.value else False
 
+    def __eq__(self, other):
+        return self._raw == other._raw
 
-def kafka_timestamp_to_datetime(timestamp):
-    return datetime.datetime.utcfromtimestamp(
-        timestamp / 1000.0
-    ) if timestamp is not None else None
-
+    def __hash__(self):
+        return hash((str(self.value), self._meta.key, self._meta.timestamp))
 
-def extract_timestamp_from_message(kafka_message):
-    timestamp_type, timestamp = kafka_message.timestamp()
-    if timestamp_type == 0 or timestamp <= 0:
-        timestamp = None
-    return timestamp
+    def __lt__(self, other):
+        return hash(self) < hash(other)
```

Comment:
This one I have a question about. Can we rely that the returned value from `hash()` will monotonically increase with the timestamp?

Reply:
Looks like it!
```python
[hash(time.time()) for t in range(0, 100)]
```
But in any case, this really doesn't make any difference. It's only used in the `sorted` function when printing the result.